libunwind/test/bad_unwind_info.pass.cpp

```diff
@@ -26,7 +26,7 @@
 __attribute__((naked)) void bad_unwind_info() {
 #if defined(__aarch64__)
   __asm__("// not using 0 because unwinder was already resilient to that\n"
-          "mov     x8, #4\n"
+          "mov     x8, #12\n"
```

Comment:
Note: changed this for an 8 byte alignment check, because `0x4 & 0x7` is `0x0`, whereas `0xC & 0x7` is `0x8`.

Reply:
Undid this with the removal of the alignment code.